usw-suwiki-user/src/main/java/usw/suwiki/domain/user/User.java

```diff
@@ -199,4 +199,8 @@ public void increaseRestrictedCountByReportedPost() {
   public void increasePointByReporting() {
     this.point += 1;
   }
+
+  public boolean isUserNextTimeBlacklistTarget() {
+    return this.restrictedCount == 2;
```

Comment:
네이밍이 약간 가독성이 떨어지는 듯한 기분!
이런 네이밍은 어때?
```suggestion
  public boolean isCloseToArrest() {
    return this.restrictedCount == ARREST_LIMIT;
```

Reply:
어레스트 좋네요 허허 저는 영어가 짧아서 ^^;;;;